apps/voting/app/src/vote-types.js

```diff
@@ -1,3 +1,7 @@
-export const VOTE_UNKNOWN = -1
-export const VOTE_NO = 0
-export const VOTE_YES = 1
+export const VOTE_ABSENT = Symbol('VOTE_ABSENT')
+export const VOTE_NAY = Symbol('VOTE_NAY')
+export const VOTE_YEA = Symbol('VOTE_YEA')
+
+export const VOTE_STATUS_ONGOING = Symbol('VOTE_STATUS_ONGOING')
+export const VOTE_STATUS_REJECTED = Symbol('VOTE_STATUS_REJECTED')
+export const VOTE_STATUS_ACCEPTED = Symbol('VOTE_STATUS_ACCEPTED')
```

Comment:
Just thinking these might look better as two object groups: `VOTE_TYPE` and `VOTE_STATUS`.